ttt_game.py

```diff
@@ -101,7 +101,7 @@ def horizontal_three_of_a_kind(self, player):
             if self.board[i][0] == self.board[i][1] == self.board[i][2] == player:
                 return True
 
-    def vertical_three_of_a_kind(self, player):
+    def is_any_column_complete(self, player):
         """
```

Comment:
Function name change MUST be accompanied by at least one test for that function if none already exist.